### PR TITLE
Better default configuration

### LINKED ISSUE #5: Cache directory issue
```sh
⏳  Fetching issues...
fs.js:894
  return binding.mkdir(pathModule._makeLong(path),
                 ^

Error: EACCES: permission denied, mkdir '//usr/local/lib/node_modules/@elao/github-agile-dashboard/cache'
    at Object.fs.mkdirSync (fs.js:894:18)
    at fullPath.split.reduce (/usr/local/lib/node_modules/@elao/github-agile-dashboard/src/Loader/Cache.js:74:20)
    at Array.reduce (native)
    at Cache.ensureDirectoryExists (/usr/local/lib/node_modules/@elao/github-agile-dashboard/src/Loader/Cache.js:70:29)
    at Cache.load (/usr/local/lib/node_modules/@elao/github-agile-dashboard/src/Loader/Cache.js:38:14)
    at HttpLoader.load (/usr/local/lib/node_modules/@elao/github-agile-dashboard/src/Loader/HttpLoader.js:49:32)
    at new GithubAgileDashboard (/usr/local/lib/node_modules/@elao/github-agile-dashboard/src/GithubAgileDashboard.js:28:21)
    at Object.<anonymous> (/usr/local/lib/node_modules/@elao/github-agile-dashboard/gad.js:31:1)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
```

Le problème est que à ma connaissance `/usr/local/lib/` n'est habituellement pas accessible en écriture par l'utilisateur, hors super user.

Plusieurs propositions d'emplacement de cache:
- le dossier utilisateur, avec un `.gad/cache` ? Pour récupérer le dossier utilisateur, utiliser la variable d'env `HOME` qui est généralement set un peu partout, y compris sous windows il me semble (ou simplement via https://nodejs.org/api/os.html#os_os_homedir).
- le "local app data" dans le dossier utilisateur:
    - Sous windows: `C:\Users\<username>\AppData\Local`
    - Sous OS X: `~/Library/Caches`
    - ...
    Mais il ne semble pas y avoir de solution générique pour l'obtenir, à part différentes variables d'env selon l'OS.
- simplement dans `/tmp` (removed à chaque reboot) ? https://nodejs.org/api/os.html#os_os_tmpdir

---

some refs: 
- https://stackoverflow.com/questions/11113974/what-is-the-cross-platform-way-of-obtaining-the-path-to-the-local-application-da
- https://nodejs.org/api/os.html
- https://stackoverflow.com/questions/19275776/node-js-how-to-get-the-os-platforms-user-data-folder

### LINKED ISSUE #6: Load Github token from git config
It seems better to avoid loading the Github Token from an environment variable.
Something like [this](https://getcomposer.org/doc/articles/troubleshooting.md#api-rate-limit-and-oauth-tokens) seems preferable: 

`composer config -g github-oauth.github.com <oauthtoken>`

Thanks @chalasr for the suggestion.

### PR DESCRIPTION
**Pull request by Tom32i**
- [x] Load Github token from git config - Fix #6 
- [x] Set default cache path in user home - Fix #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,29 +8,30 @@
 
 ## Configuration
 
-### GitHub Access Token
+### GitHub authentication
 
-[Generate](https://github.com/settings/tokens) a [GitHub personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/) with the following access rights: `repo` and `user`.
-Edit your bash file with `sudo vim ~/.bashrc` and export the generated token as below:
+As [recommended by GitHub](https://github.com/blog/180-local-github-config), Gad rely on `github.user` and `github.token` variables in your git config to authenticate you on GitHub API. Follow this three steps and you're good to go:
```
Comment (chalasr) [inline README.md]: rely => relies

File: README.md [hunk 2]
```diff
@@ -54,6 +55,14 @@ You can manually specify any of the options on the fly:
 
 E.g.: `gad -o Elao -r symfony-standard -u Tom32i -p MyGitHubPersonalAccessToken`
 
+#### Cache
+
+For performance purpose, Gad keeps a little cache file for each repository.
+
+By default, it's located in `~/.gad/cache` but you can specify a different path with the `cacheDir`option:
```
Comment (chalasr) [inline README.md]: missing space before "option"

File: README.md [hunk 3]
```diff
@@ -54,6 +55,14 @@ You can manually specify any of the options on the fly:
 
 E.g.: `gad -o Elao -r symfony-standard -u Tom32i -p MyGitHubPersonalAccessToken`
 
+#### Cache
```
Comment (chalasr) [inline README.md:58]: This title is smaller than others, is it expected?

**Review by chalasr — APPROVED** [review]
👌 left some minor comments

File: README.md (hunk 1 above)
Comment (chalasr) [reply README.md]: This three steps => Those|These

File: README.md (hunk 3 above)
Comment (Tom32i) [reply README.md:58]: Yeah I meant it as a sub-section of __Options__. What do you think?

Comment (chalasr) [reply README.md:58]: 👍  sorry

**Review by chalasr — APPROVED** [review]
👍

**Review by ogizanagi — APPROVED** [review]

**Pull request merged**